services/expiration/expiration-notification.server.ts

```diff
@@ -173,16 +160,7 @@ const getExpiringDnsRecords = () => {
       expiresAt: {
         lte: dayjs().add(30, 'd').toDate(),
       },
-      OR: [
-        {
-          lastNotified: null,
-        },
-        {
-          lastNotified: dayjs()
-            .subtract(NOTIFICATION_FREQUENCY * 4, 'd')
-            .toDate(),
-        },
-      ],
+      lastNotified: null,
```

Comment:
I think this is the right fix for certs, since "renew" means "create a new record."  But with DNS records, we might renew multiple times.

Should we set the `lastNotified` to `null` again when we renew a DNS Record?  Then this would work I think.

Reply:
No, that would send another notification.

Reply:
I have modified the condition to check if the date is set to a month in the past (like we were doing before).

Reply:
Yes, this works. I tested it locally.